dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/CompanyDataControllerTest.kt

```diff
@@ -200,4 +202,26 @@ class CompanyDataControllerTest {
             "The exception message does not say that a 403 client error was the cause.",
         )
     }
+
+    @Test
+    fun `post a dummy company twice and receive the expected error code`() {
+        val testCompanyInformation = apiAccessor.testDataProviderForEuTaxonomyDataForNonFinancials
+            .getCompanyInformationWithoutIdentifiers(1).first()
+        val randomIsin = CompanyIdentifier(
+            identifierValue = UUID.randomUUID().toString(),
+            identifierType = CompanyIdentifier.IdentifierType.isin,
+        )
+        val randomizedCompanyInformation = testCompanyInformation.copy(identifiers = listOf(randomIsin))
+        apiAccessor.tokenHandler.obtainTokenForUserType(UserType.Uploader)
```

Comment:
After merging to master apiAcessor.tokenHandler and UserType are no longer defined. They have been removed as part of DALA-394:
https://github.com/d-fine/Dataland/pull/160
https://github.com/d-fine/Dataland/pull/160/files#diff-6b14d301d98423aa73b99ca9e10a92a27a895ed7caf7cb6181208df6e3a0c39c

Reply:
Should be fixed in the current version